doc/source/whatsnew/v1.1.0.rst

```diff
@@ -109,6 +109,7 @@ Datetimelike
 - :meth:`DatetimeArray.searchsorted`, :meth:`TimedeltaArray.searchsorted`, :meth:`PeriodArray.searchsorted` not recognizing non-pandas scalars and incorrectly raising ``ValueError`` instead of ``TypeError`` (:issue:`30950`)
 - Bug in :class:`Timestamp` where constructing :class:`Timestamp` with dateutil timezone less than 128 nanoseconds before daylight saving time switch from winter to summer would result in nonexistent time (:issue:`31043`)
 - Bug in :meth:`DataFrame.reindex` and :meth:`Series.reindex` when reindexing with a tz-aware index (:issue:`26683`)
+- Bug in :meth: `read_csv` used in file like object `RawIOBase` is not recognize `encoding` option (:issue:`31575`)
```

Comment:
Does this render correctly? (see [building the documentation](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#building-the-documentation))

I think you might need 
```
:meth:`read_csv`
```
(without the blank space)

Reply:
Oh, sorry. And change log position is not correct.